test-suite/src/ordering.rs

```diff
@@ -35,7 +35,7 @@ test_case!(ordering, async move {
         (0, "SELECT * FROM Operator WHERE 3 > 3;"),
         (
             5,
-            "SELECT * FROM Operator o1 WHERE 3 > (SELECT id FROM Operator WHERE o1.id < 100);",
+            "SELECT * FROM Operator o1 WHERE 3 > (SELECT min(id) FROM Operator WHERE o1.id < 100);",
```

Comment:
nit.
All other SQL keywords are using uppercase letters but `min` does not though `min` is also the SQL keyword.
It would be good to use all uppercase for consistency.